service/src/main/java/co/cask/wrangler/service/kafka/KafkaService.java

```diff
@@ -281,12 +282,11 @@ public void specification(HttpServiceRequest request, final HttpServiceResponder
 
       Map<String, String> properties = new HashMap<>();
       properties.put("topic", topic);
-      properties.put("referenceName", topic);
+      properties.put("referenceName", topic + "_" + UUID.randomUUID());
```

Comment:
it clashes with the kafka offset table name. It defaults to the topic name.

We can't use a hard coded offset table name, because then every time you create a pipeline it uses the same offset table, so offsets consumed by a pipeline affect those of the other pipeline. 

One thing we could do is keep reference name as topic only, and make the offset table name unique. Would that work?